nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -151,8 +151,10 @@ private static void reportInvalidReturnTypeError(
   }
 
   /**
-   * This method returns type of the tree considering that the parameterized typed tree annotations
-   * are not preserved if obtained directly using ASTHelpers.
+   * This method returns the type of the tree, including any type use annotations. This is required
```

Comment:
I would still introduce a paragraph break after the first line here. Since the first line is a description of what the method does and the rest is additional explanation. It might need a `<p>` at the beginning of the new paragraph in order for GJF to not reformat it...

Reply:
Fixed in [f000611](https://github.com/uber/NullAway/pull/734/commits/f00061117e89e34b8b8bff89bdd21ca15e5f5e2a)